.github/workflows/unit_tests.yml

```diff
@@ -109,6 +109,15 @@ jobs:
         run: py.test --cov=./ --cov-report=xml
       - name: Upload test coverage to Codecov
         uses: codecov/codecov-action@v1
+        # Build and publish dbdocs
+      - name: Download dbdocs
+        run: npm install -g dbdocs
+      - name: Generate DBML
+        run: python manage.py dbml > cdh-web.dbml
+      - name: Login to dbdocs
+        run: echo ${{ secrets.DBDOCS_KEY }} | dbdocs login
+      - name: Push to dbdocs
+        run: dbdocs build cdh-web.dbml --project cdh-web
```

Comment:
Can we do `cdhweb` instead of `cdh-web` for the project name? Nick and I just refactored the deploy scripts to use the name without the hyphen, we both find it more readable.

Reply:
✅ addressed with fad2128